src/sentry/seer/explorer/client.py

```diff
@@ -50,237 +21,277 @@
+        prompt: str,
+        insert_index: int | None = None,
+        on_page_context: str | None = None,
+    ) -> int:
+        """
+        Continue an existing Seer Explorer session.
+
+        This allows you to add follow-up queries to an ongoing conversation.
+        User context is NOT collected again (it was already captured at start).
+
+        Args:
+            run_id: The run ID from start_run()
+            prompt: The follow-up task/query for the agent
+            insert_index: Optional index to insert the message at
+            on_page_context: Optional context from the user's screen
+
+        Returns:
+            int: The run ID (same as input)
+
+        Raises:
+            requests.HTTPError: If the Seer API request fails
+        """
+        path = "/v1/automation/explorer/chat"
+
+        payload: dict[str, Any] = {
+            "organization_id": self.organization.id,
+            "query": prompt,
+            "run_id": run_id,
+            "insert_index": insert_index,
+            "on_page_context": on_page_context,
+        }
+
+        body = orjson.dumps(payload, option=orjson.OPT_NON_STR_KEYS)
+
+        response = requests.post(
+            f"{settings.SEER_AUTOFIX_URL}{path}",
+            data=body,
+            headers={
+                "content-type": "application/json;charset=utf-8",
+                **sign_with_seer_secret(body),
+            },
+        )
+
+        response.raise_for_status()
+        result = response.json()
+        return result["run_id"]
+
+    def get_run(
+        self,
+        run_id: int,
+        blocking: bool = False,
+        poll_interval: float = 2.0,
+        poll_timeout: float = 600.0,
+    ) -> SeerRunState:
+        """
+        Get the status/result of a Seer Explorer session.
+
+        If artifact_schema was provided in the constructor and an artifact was generated,
+        it will be automatically reconstructed as a typed Pydantic instance.
+
+        Args:
+            run_id: The run ID returned from start_run()
+            blocking: If True, blocks until the run completes (with polling)
+            poll_interval: Seconds between polls when blocking=True
+            poll_timeout: Maximum seconds to wait when blocking=True
+
+        Returns:
+            SeerRunState: State object with blocks, status, and optionally reconstructed artifact
+
+        Raises:
+            requests.HTTPError: If the Seer API request fails
+            TimeoutError: If polling exceeds poll_timeout when blocking=True
+        """
+        if blocking:
+            state = poll_until_done(run_id, self.organization, poll_interval, poll_timeout)
+        else:
+            state = fetch_run_status(run_id, self.organization)
+
+        # Automatically parse raw_artifact into typed artifact if schema was provided
+        if state.raw_artifact and self.artifact_schema:
+            try:
+                state.artifact = self.artifact_schema.parse_obj(state.raw_artifact)
```

Comment:
```suggestion
                state.artifact = self.artifact_schema.parse_obj(state.raw_artifact)
                state.raw_artifact = None
```
maybe? to avoid returning dup data in api